### DISCUSSION TITLE
Desktop browser messaging

### DISCUSSION THREAD

**Discussion by KaKi87** [FossifyOrg/General-Discussion#502] [Ideas]
Hi,

It would be nice to be able to read and write texts from a desktop browser like messages.google.com does, with notifications and multi-SIM support.

Thanks

**Comment by naveensingh** [comment]
It would be nice but that would require access to the internet and no Fossify apps are allowed that access ([see why](https://github.com/orgs/FossifyOrg/discussions/68#discussioncomment-8001994)).

We might solve this in the future with more flavors but for now, there are no plans to include any internet related features.
